packages/ui/dropdown/src/dropdown-menu-item.tsx

```diff
@@ -59,19 +65,68 @@ const DropdownMenuItem: FC<DropdownMenuItemProps> = ({
 		IconTag = Icons[icon]
 	}
 
+	const getContent = () => {
+		let content = <span id={menuTitleId}>{children}</span>
+
+		switch (_type) {
+			case "select":
+				break
+			case "select-checkbox":
+				content = (
+					<Checkbox
+						label={content}
+						isSmall={true}
+						isChecked={isChecked}
+						isDisabled={isDisabled}
+						{..._renderHTMLPropsSafely(_checkboxProps)}
+					/>
+				)
+				break
+			case "select-variable":
+				IconTag = IconTag ? IconTag : Icons.Add
+				content = (
+					<>
+						<div id={menuTitleId} className="sui-dropdown__menu-item--wrapper">
+							<div>
+								<IconTag size="sm" className="sui-dropdown__menu-item-icon" />
+							</div>
+							<div>
+								<span className="sui-dropdown__menu-item--title">
+									{children}
+								</span>
+								{variable && (
+									<span className="sui-dropdown__menu-item--var">{` {${variable}}`}</span>
```

Comment:
Make the variable param accept React.ReactNode values and remove curly braces, so if a dev wants to pass any custom DOM, they can.